contracts/interfaces/IIporOracle.sol

```diff
@@ -51,6 +51,27 @@ interface IIporOracle {
     /// @param indexValues new IPOR Index values
     function updateIndexes(address[] memory assets, uint256[] memory indexValues) external;
 
+    /// @notice Updates the IPOR index and quasi IBT (Interest Bearing Token) price for a specific asset.
+    /// @dev This function can only be called by an authorized updater and when the contract is not paused.
+    ///         It performs several checks to ensure the integrity of the update, such as asset support verification
+    ///         and timestamp validation. If any of these checks fail, the function reverts with an appropriate error message.
+    /// @param asset The address of the underlying asset for which the index and quasi IBT price are being updated.
+    /// @param indexValue The new value of the IPOR index for the specified asset.
+    /// @param updateTimestamp The timestamp at which the index value is updated.
+    ///         This value must be greater than the last update timestamp and less than the current block timestamp.
+    /// @param newQuasiIbtPrice The new quasi IBT price for the specified asset.
```

Comment:
Add detailed information what exactly is quasiIbtPrice or add reference to IporOracleTypes.IPOR doc where is more details